src/transformers/models/beit/modeling_beit.py

```diff
@@ -659,6 +659,10 @@ def forward(
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
     ) -> Union[tuple, BeitModelOutputWithPooling]:
+        r"""
+        bool_masked_pos (`torch.BoolTensor` of shape `(batch_size, num_patches)`, *optional*):
+                    Boolean masked positions. Indicates which patches are masked (1) and which aren't (0).
```

Comment:
Is this overindented or no? (not familiar with autodocstring magic logic)

Reply:
It is! I'd just lazily copy-pasted, so I've fixed here and in the original place